qa/rspec/centos/commands.rb

```diff
@@ -0,0 +1,58 @@
+# encoding: utf-8
+module ServiceTester
+  class CentosCommands
```

Comment:
This file was renamed to be redhat, as is really the _source_ of this kind of distros. I think you should rebase this PR, so you get all this things merged for you.